specification/batch/resource-manager/Microsoft.Batch/stable/2022-01-01/BatchManagement.json

```diff
@@ -2772,7 +2861,7 @@
           "x-nullable": true,
           "type": "integer",
           "format": "int32",
-          "title": "The low-priority core quota for the Batch account.",
+          "title": "The Spot/Low-priority core quota for the Batch account.",
```

Comment:
Let's switch "Low-priority" to "low-priority" to be consistent with casing in the docs (Spot is always capitalized, though)